### PR TITLE
Update Watchdog in the services list section

### PR DESCRIPTION
**Pull request by urseberry**
### What does this PR do?
Update the Watchdog in the services list section of the main Watchdog page. Content provided by Brooke Chen.

### Motivation
Watchdog docs have been unattended for a long time and are out of date.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by crunchypistachiobutter** [comment]
LGTM! Thanks

File: content/en/watchdog/_index.md [hunk 1]
```diff
@@ -48,17 +48,21 @@ Watchdog looks for irregularities in metrics, like a sudden spike in the hit rat
...
+When Watchdog detects an irregularity in an APM metric, the pink Watchdog binoculars icon appears next to the impacted service in the [APM services list][12]. The number next to the binoculars indicates the number of issues Watchdog has detected within that service.
 
-{{< img src="watchdog/service_list.png" alt="Watchdog service list" style="width:75%;" >}}
+{{< img src="watchdog/service_list.png" alt="Screenshot of the APM services list page, showing 5 services. A pink binoculars icon follows the name of the web-store service." style="width:75%;" >}}
 
-If Watchdog has discovered something out of the ordinary in a specific service, viewing the corresponding [Service page][12] reveals a dedicated Watchdog section in the middle of the page, between the application performance graphs and the latency distribution section. The Watchdog section displays any relevant Watchdog alerts.
+You can see greater detail about a metric anomaly by navigating to the [service page][13]. On the top of the service page is the Watchdog Insights box. Watchdog Insights helps you discover tag values that are associated with anomalous behaviors, such as higher error rate or latency.
```
Comment (alai97) [inline content/en/watchdog/_index.md:55]:
```suggestion
You can see greater detail about a metric anomaly by navigating to the [Services page][13]. On the top of the page is the Watchdog Insights box. Watchdog Insights helps you discover tag values that are associated with anomalous behaviors, such as higher error rate or latency. 
```

File: content/en/watchdog/_index.md [hunk 2]
```diff
@@ -48,17 +48,21 @@ Watchdog looks for irregularities in metrics, like a sudden spike in the hit rat
...
 
-{{< img src="watchdog/service_list.png" alt="Watchdog service list" style="width:75%;" >}}
+{{< img src="watchdog/service_list.png" alt="Screenshot of the APM services list page, showing 5 services. A pink binoculars icon follows the name of the web-store service." style="width:75%;" >}}
 
-If Watchdog has discovered something out of the ordinary in a specific service, viewing the corresponding [Service page][12] reveals a dedicated Watchdog section in the middle of the page, between the application performance graphs and the latency distribution section. The Watchdog section displays any relevant Watchdog alerts.
+You can see greater detail about a metric anomaly by navigating to the [service page][13]. On the top of the service page is the Watchdog Insights box. Watchdog Insights helps you discover tag values that are associated with anomalous behaviors, such as higher error rate or latency. 
 
-{{< img src="watchdog/watchdog_story_bis.png" alt="Watchdog story bis" style="width:75%;">}}
+Scrolling down on the service page, you may find the Watchdog icon on metric graphs.
```
Comment (alai97) [inline content/en/watchdog/_index.md:57]:
```suggestion
You can also find the Watchdog icon on metric graphs.
```

File: content/en/watchdog/_index.md [hunk 3]
```diff
@@ -48,17 +48,21 @@ Watchdog looks for irregularities in metrics, like a sudden spike in the hit rat
...
+
+{{< img src="watchdog/latency_graph.png" alt="A graph showing service latency, in seconds, on the y-axis and the time of day on the x-axis. The entire graph is highlighted in pink, and the words May 2: 13:31 Ongoing appear at the top" style="width:75%;" >}}
+
+Clicking on the binoculars icon brings up a [Watchdog alert][14] card with additional details.
```
Comment (alai97) [inline content/en/watchdog/_index.md:61]:
```suggestion
Click on the binoculars icon to see a [Watchdog alert][14] card with more details.
```

**Review by alai97 — APPROVED** [review]
Looks good, three copy nits!

**Pull request merged**